tests/datasource/data_connector/test_data_connector.py

```diff
@@ -52,38 +52,6 @@ def basic_data_connector(tmp_path_factory):
     return basic_data_connector
 
 
-def test_get_batch_definition_list_from_batch_request():
```

Comment:
This is old code and these aren't tests so I've deleted them.

Reply:
👍🏼